adoc/admin-updates.adoc

```diff
@@ -101,6 +101,68 @@ The control plane consists of these components:
 * cri-o
 =====
 
+==== Applying an update
+
+[NOTE]
+=====
+The following steps are temporary and will be integrated into the official product soon.
+=====
+
+Updates have to be applied on a per-node base, starting from the control plane, down to the worker nodes.
+
+. To prepare the update, `ssh` to each of the nodes and perform the following steps:
+.. Disable the `skuba-update.timer`:
++
+----
+sudo systemctl disable --now skuba-update.timer
+----
+.. Create a temporary lock on the kubernetes component system packages:
```

Comment:
I didn't need to do this in my environment, I did `zypper in kubernetes-kubeadm=1.15.0` and it worked just fine, just had to confirm that I wanted to "break" `1.15.0` when asked by zypper and I had `kubeadm=1.15.0` and the rest on `1.14.1`.

If that works I think it's even easier than asking to create locks.